spec/support/system_helpers.rb

```diff
@@ -40,4 +40,16 @@ def resize_window(width: nil, height: nil)
   ensure
     page.driver.browser.manage.window.resize_to(original_size.width, original_size.height)
   end
+
+  def using_browser_timezone(timezone, &example)
```

Comment:
Wow this is cool, I didn't know about `using_session` and I also didn't know we had the TZ in env!